src/reducers/breakpoints.js

```diff
@@ -176,55 +155,101 @@ function addBreakpoint(state, action) {
 }
 
 function removeBreakpoint(state, action) {
-  if (action.status === "done") {
-    const id = makeLocationId(action.breakpoint.location);
+  if (action.status != "done") {
+    return state;
+  }
 
-    if (action.disabled) {
-      const bp = state.breakpoints.get(id);
-      const updatedState = state.setIn(
-        ["breakpoints", id],
-        updateObj(bp, {
-          loading: false,
-          disabled: true
-        })
-      );
+  const id = makeLocationId(action.breakpoint.location);
+  const pendingId = makePendingLocationId(action.breakpoint.location);
+  let updatedState = undefined;
+
+  if (action.disabled) {
```

Comment:
since these two cases are so different, perhaps they can be functions that return `updatedState`? then we won't need to do the weird `let`